pkg/helm/client.go

```diff
@@ -56,7 +56,7 @@ type CliClient struct {
 }
 
 // findHelmVersion returns HelmVersion based on helm binary present in the path
-func findHelmVersion() (HelmVersion, error) {
+func FindHelmVersion() (HelmVersion, error) {
```

Comment:
Let's rename this function to `FindVersion()` which can be used as `helm.FindVersion()` after import.